dummy module error

Hi,

I am trying to generate a new project and I selected these options:
Rest API
No database
No CI/CD
Add dummy model                                                                                                                                       
Add example routers
Adds traefik labels to docker container  

I am getting an error since the dummy module is missing. Am I doing something wrong? I tried running it with Docker too same exception. Great project btw.

Mac Ventura 13.3
Python 3.11.4
Poetry (version 1.5.1)
fastapi==0.97.0
fastapi-template==4.1.5

Error:

```
Check with Flake8........................................................Passed
Validate types with MyPy.................................................Failed
- hook id: mypy
- exit code: 1

deriv_project/web/api/router.py:3: error: Module "deriv_project.web.api" has no
attribute "dummy"  [attr-defined]
    from deriv_project.web.api import dummy, echo, monitoring
    ^
Found 1 error in 1 file (checked 18 source files)

Check python ast.........................................................Passed
Trim Trailing Whitespace.................................................Passed
Check Toml...............................................................Passed
Fix End of Files.........................................................Passed
Add trailing commas......................................................Passed
Pretty format YAML.......................................................Passed
Format with Black........................................................Passed
autoflake................................................................Passed
isort....................................................................Passed
Check with Flake8........................................................Passed
Validate types with MyPy.................................................Failed
- hook id: mypy
- exit code: 1

deriv_project/web/api/router.py:3: error: Module "deriv_project.web.api" has no
attribute "dummy"  [attr-defined]
    from deriv_project.web.api import dummy, echo, monitoring
    ^
Found 1 error in 1 file (checked 18 source files)

Project successfully generated. You can read information about usage in README.md
```

--- a/fastapi_template/cli.py
+++ b/fastapi_template/cli.py
@@ -380,6 +380,7 @@ features_menu = MultiselectMenuModel(
             code="add_dummy",
             cli_name="dummy",
             user_view="Add dummy model",
+            is_hidden=lambda ctx: ctx.orm == "none",
             description=(
                 "This option creates {what} as an example of how to use chosen ORM.\n"
                 "Also this option will generate you an example of {dao}.".format(

